bin/main.rb

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env ruby
+
+require '../app'
+
+class Main
+  attr_reader :app
+
+  def initialize
+    puts 'Welcome to School Library App!'
+    @app = App.new
+    show_menu
+  end
+
+  def show_menu
+    puts ''
+    puts "Please choose an option by entering a number:
+  1 - List all books
+  2 - List all people
+  3 - Create a person
+  4 - Create a book
+  5 - Create a rental
+  6 - List all rentals for a given person id
+  7 - Exit"
+    user_selection
+  end
+
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/MethodLength
+  def user_selection
+    selection = gets.chomp.to_i
+    case selection
+    when 1
+      app.list_all_books
+      show_menu
+    when 2
+      app.list_all_people
+      show_menu
+    when 3
+      print 'Do you want to create a student (1) or a teacher (2)? [Input the number]:'
```

Comment:
- [ ] **REQUIRED:** WIth one method to create the person you will not need to nest switch to create a Teacher or a Student, your code should be clean. Please consider having only one switch to create the person